### PR TITLE
updated context for wishlist

### PR DESCRIPTION
**Pull request by vidhi-thakur**
- added context for wishlist 
- updated values in different component for wishlist context

### CODE REVIEW

**Review by vikrantkode — APPROVED** [review]
- Code is clean 
- comments are good for reviewer and u put at some places..its good
- good to merge

**Review by PrashanthreddyAinala — APPROVED** [review]
Code looks clean good to merge.

File: src/components/UIcomponents/noDataPage/NoData.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.noData {
+    margin: 5rem auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
```
Comment (Mahi24-prog) [inline src/components/UIcomponents/noDataPage/NoData.css:6]: use utility classes for flex

File: src/components/cart-management/cart-management-card/CartManagementCard.js [hunk 2]
```diff
@@ -19,6 +21,22 @@ function CartManagementCard({ id, title, author, rating, description, price, pro
...
+            type: "ADD_TO_WISHLIST",
+            payload: {
+                title,
+                author,
+                rating,
+                description,
+                price,
+                productImage,
+                id
+            }
```
Comment (Mahi24-prog) [inline src/components/cart-management/cart-management-card/CartManagementCard.js]: you can send only product id in payload. It not good practice to share whole product data.

File: src/context/addToWishlist.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+const WishlistContext = createContext();
+
+const useWishlist = () => useContext(WishlistContext)
+
+const WishlistProvider = ({ children }) => {
+    return (
+        <WishlistContext.Provider value={useReducer(wishlistReducer, initialWishlist)}>
+            {children}
+        </WishlistContext.Provider>
```
Comment (Mahi24-prog) [inline src/context/addToWishlist.js:13]: you are calling useReducer in values, it not good practice. Call it saperately on top.

**Review by Mahi24-prog — APPROVED** [review]
Check suggestions I have given on code. Try to improve it.

**Pull request merged**